gradle/plugins/src/main/kotlin/org/cqfn/diktat/buildutils/spotless-convention-configuration.gradle.kts

```diff
@@ -0,0 +1,39 @@
+package org.cqfn.diktat.buildutils
```

Comment:
let's remove spotless from diktat, since we don't use it in maven too.
also, need to create a repro with spotless plugin and module with similar name as dependency to plugin